app/grants/admin.py

```diff
@@ -55,8 +56,9 @@ class GrantAdmin(GeneralAdmin):
         'logo_svg_asset', 'logo_asset',
         'team_member_list',
         'subscriptions_links', 'contributions_links', 'link',
+        'migrated_to'
     ]
-    list_display =['pk', 'title', 'active','grant_type', 'link', 'hidden']
+    list_display =['pk', 'title', 'active','grant_type', 'link', 'hidden', 'migrated_to']
     raw_id_fields = ['admin_profile']
```

Comment:
should we add to raw_id_fields? i honestly just like the raw id field explorer more than the regular FK explorer

Reply:
You mean it'd end up like this -> I didn't want this to be edited via admin cause I wanted to ensure this is done via migrate_grant.py script file 


right now -> the `migrated_to` displayed in table and within the grant admin page is a read only mode 

<img width="634" alt="Screenshot 2020-01-16 at 2 39 27 PM" src="https://user-images.githubusercontent.com/5358146/72509757-3aaa0380-386e-11ea-8af2-242920b5dd11.png">